datacapturing/src/main/java/de/cyface/datacapturing/ReconnectCallback.java

```diff
@@ -78,7 +79,7 @@ public void timedOut() {
         lock.lock();
         try {
             if (!wasRunning()) {
-                Log.w(TAG, "Unable to bind on reconnect. It seems the service is not running");
+                Log.w(TAG, "Reconnect called while background service was not running. Capturing seems to be paused or stopped.");
```

Comment:
We should discuss how we need to change reconnect and isRunning so that we avoid calling isRunning twice.

Reply:
this is a call to `wasRunning` which just fetches the attribute. That should be fast and ok to do.